core/src/test/scala/unit/kafka/admin/ResetConsumerGroupOffsetTest.scala

```diff
@@ -26,43 +26,10 @@ import org.apache.kafka.test
 import org.junit.Assert._
 import org.junit.Test
 
-import scala.jdk.CollectionConverters._
```

Comment:
same as above.

Reply:
@brary  Hi Navi They are indent automatically by Intellij can you please help me with Intellij Code Style XML so that i will indent it accordingly.

Reply:
You can manually cut/paste the changed imports back. It will revert them to the original commit.